### PR TITLE
demo/billing: default to creating a topic always in Docker

### LINKED ISSUE #4765: Billing demo generates no events
### What version of Materialize are you using?

tags/v0.5.1

### How did you install Materialize?

* [ ] Docker image
* [ ] Linux release tarball
* [ ] APT package
* [ ] macOS release tarball
* [ ] Homebrew tap
* [X] Built from source

### What was the issue?

I am following the steps to [run the demo](https://materialize.com/docs/demos/microservice/#run-the-demo), but I never get events in the database.

### Is the issue reproducible? If so, please provide reproduction instructions.

```
git clone https://github.com/MaterializeInc/materialize.git
git checkout v0.5.1
cd ./materialize/demo/billing
./mzcompose up
```
All the services startup then I keep seeing the error message in the logs show in the section below. I have sources in my database, but no events.

```
materialize=> SHOW SOURCES;
      name
----------------
 billing_source
 price_source
(2 rows)

materialize=> SELECT jsonb_pretty(records::jsonb) AS records FROM billing_raw_data LIMIT 1;
 records
---------
(0 rows)
```

I am running
* Docker version 20.10.0-beta1, build ac365d7
* docker-compose version 1.27.4, build 40524192
* macOS Catalina 10.15.7.

### Please attach any applicable log files.

`materialized_1                | Nov 11 22:41:45.086 ERROR rdkafka::client: librdkafka: Global error: UnknownTopicOrPartition (Broker: Unknown topic or partition): billing [0]: topic does not exist (Broker: Unknown topic or partition)`

### PR DESCRIPTION
**Pull request by ruchirK**
Fix #4765.

Update the docs to use mzconduct instead of mzcompose and add a new workflow for testing locally.

---
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/4778)

### CODE REVIEW

**Comment by ruchirK** [comment]
@benesch I ended up having to create a new workflow for local testing. when I tried to do `bin/mzconduct up billing` today it had the same problem as with using mzcompose directly. Not entirely sure what changed because last night I thought `bin/mzconduct up billing` was working -- I may have messed something up then.

Let me know if you think the extra workflow is not a good solution and I'll dig into it more

**Review by benesch — APPROVED** [review]
Seems reasonable to me!

File: demo/billing/mzcompose.yml [hunk 1]
```diff
@@ -84,6 +87,13 @@ volumes:
 
 mzconduct:
   workflows:
+    local:
```
Comment (quodlibetor) [inline demo/billing/mzcompose.yml:90]: I'd consider renaming this to `demo` to make it obvious to future developers that this is what shows up in the docs.

**Comment by quodlibetor** [comment]
cc #4769

**Pull request merged**